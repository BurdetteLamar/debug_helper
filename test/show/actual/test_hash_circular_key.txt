---
Hash (size=2 name='test_hash_circular_key'):
  Pair 0:
    Key:
      Hash (size=0): {}
    Value: Fixnum 0
  Pair 1:
    Key: Hash {{}=>0, {...}=>0}
    Value: Fixnum 0
