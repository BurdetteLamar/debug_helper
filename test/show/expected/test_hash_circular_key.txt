---
Hash (message='test_hash_circular_key'):
  size: 2
  default: 
  default_proc: 
  Pair 0:
    Key:
      Hash:
        size: 0
        default: 
        default_proc: 
    Value: Fixnum 0
  Pair 1:
    Key: Hash {{}=>0, {...}=>0}
    Value: Fixnum 0
